content/HowItWorks/DeviceTypes.md

```diff
@@ -6,110 +6,178 @@ weight = 10
-  - [Occupancy Sensor](#occupancy-sensor)
-  - [Temperature Sensor](#temperature-sensor)
-  - [Pressure Sensor](#pressure-sensor)
-  - [Flow Sensor](#flow-sensor)
-  - [Humidity Sensor](#humidity-sensor)
-  - [On/Off Sensor](#onoff-sensor)
-  - [Smoke CO Alarm](#smoke-co-alarm)
-- Closures
-  - [Door Lock](#door-lock)
-  - [Door Lock Controller](#door-lock-controller)
-  - [Window Covering](#window-covering)
-  - [Window Covering Controller](#window-covering-controller)
-- HVAC
-  - [Heating/Cooling Unit](#heatingcooling-unit)
-  - [Thermostat](#thermostat)
-  - [Fan](#fan)
-  - [Air Purifier](#air-purifier)
-  - [Air Quality Sensor](#air-quality-sensor)
-- Media
-  - [Basic Video Player](#basic-video-player)
-  - [Casting Video Player](#casting-video-player)
-  - [Speaker](#speaker)
-  - [Content App](#content-app)
-  - [Casting Video Client](#casting-video-client)
-  - [Video Remote Control](#video-remote-control)
-- Robotic Devices
-  - [Robotic Vacuum Cleaner](#robotic-vacuum-cleaner)
-- Appliances
-  - [Refrigerator](#refrigerator)
-  - [Temperature Controlled Cabinet](#temperature-controlled-cabinet)
-  - [Room Air Conditioner](#room-air-conditioner)
-  - [Laundry Washer](#laundry-washer)
-  - [Dishwasher](#dishwasher)
-
-### On/Off Light
+As of Matter 1.3 the following types are certifiable.
+
+- [Utility Device Types](#utility-device-types)
+  - [Root Node](#root-node)
+  - [Power Source](#power-source)
+  - [OTA Requestor](#ota-requestor)
+  - [OTA Provider](#ota-provider)
+  - [Bridged Node](#bridged-node)
+  - [Electrical Sensor](#electrical-sensor)
+  - [Device Energy Management](#device-energy-management)
+- [Application Device Types](#application-device-types)
+  - [Lighting](#lighting)
+    - [On/Off Light](#onoff-light)
+    - [Dimmable Light](#dimmable-light)
+    - [Color Temperature Light](#color-temperature-light)
+    - [Extended Color Light](#extended-color-light)
+  - [Smart Plugs/Outlets and Other Actuators](#smart-plugsoutlets-and-other-actuators)
+    - [On/Off Plug in Unit](#onoff-plug-in-unit)
+    - [Dimmable Plug in Unit](#dimmable-plug-in-unit)
+    - [Pump](#pump)
+    - [Water Valve](#water-valve)
+  - [Switches and Controls](#switches-and-controls)
+    - [On/Off Light Switch](#onoff-light-switch)
+    - [Dimmer Switch](#dimmer-switch)
+    - [Color Dimmer Switch](#color-dimmer-switch)
+    - [Control Bridge](#control-bridge)
+    - [Pump Controller](#pump-controller)
+    - [Generic Switch](#generic-switch)
+  - [Sensors](#sensors)
+    - [Contact Sensor](#contact-sensor)
+    - [Light Sensor](#light-sensor)
+    - [Occupancy Sensor](#occupancy-sensor)
+    - [Temperature Sensor](#temperature-sensor)
+    - [Pressure Sensor](#pressure-sensor)
+    - [Flow Sensor](#flow-sensor)
+    - [Humidity Sensor](#humidity-sensor)
+    - [On/Off Sensor](#onoff-sensor)
+    - [Smoke CO Alarm](#smoke-co-alarm)
+    - [Water Freeze Detector](#water-freeze-detector)
+    - [Water Leak Detector](#water-leak-detector)
```

Comment:
Are the water Freeze Detector and Rain Sensor certifiable device types? It looks to me that they use the same clusters (Boolean State) as Water Leak detector ? So did these therefore also get through 1.3?

Reply:
That is certainly my impression, we are testing clusters at certification and not device types (for now) and the clusters did land as a part of 1.3.